doc/cortx-minikube-k8s-installation.md

```diff
@@ -0,0 +1,295 @@
+    [started]  hax        0x7200000000000001:0x7   inet:tcp:cortx-data-headless-svc-car@22001
+    [started]  ioservice  0x7200000000000001:0xa   inet:tcp:cortx-data-headless-svc-car@21001
+    [started]  ioservice  0x7200000000000001:0x17  inet:tcp:cortx-data-headless-svc-car@21002
+    [started]  confd      0x7200000000000001:0x24  inet:tcp:cortx-data-headless-svc-car@21003
+
+```
+
+*Note: It may take several minutes for s3server instances to move from "offline" to "started"*
+
+*If the pods are not coming up correctly or any of the service are not getting `[started]` - check your `solution.yaml` for typos or mistakes which could result in a deployment failure.*
+
+**5.5 Destroy CORTX Cluster**
+
+To rollback to step 5.3 and destroy the CORTX cluster run the below command:
+
+```
+./destroy-cortx-cloud.sh
+```
+
+**6. Using CORTX**
+
+Use CORTX CSM (Management API) to provision an S3 account
+
+**6.1 Login to the management**
+
+```
+export CSM_IP=`kubectl get svc cortx-control-loadbal-svc  -ojsonpath='{.spec.clusterIP}'`
+
+curl rl -v -d '{"username": "cortxadmin", "password": "Cortxadmin@123"}' https://$CSM_IP:8081/api/v2/login --insecure
+```
+
+*Copy the bearer token for the next command*
+
+**6.2 Create an S3 account.**
+
+```
+curl -H 'Authorization:  Bearer <bearer-token>' -d '{  "account_name": "testUser",   "account_email": "*****@gmail.com",   "password": "Account@1" }' https://$CSM_IP:8081/api/v2/s3_accounts --insecure
+```
+
+- Results from the above command
+
+```
+{"account_name": "testUser", "account_email": "*****@gmail.com", "account_id": "507040439091", "canonical_id": "92b845b0de8a4532a0d3a15a1540e43ffc1ec5a02662430ea76dce69d3e770fb", "access_key": "AKIA*******************CKw", "secret_key": "U1pU****************************cGL"}
+```
+
+**7.  Install and configure AWS CLI to use IAM and S3 APIs**
+
+```
+sudo yum install -y unzip
+curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+unzip awscliv2.zip
+sudo ./aws/install
+```
+
+*Your credentials are what you got after creating an s3 account above.*
+
+```
+export AWS_ACCESS_KEY_ID=AKIA*******************CKw
+export AWS_SECRET_ACCESS_KEY=U1pU****************************cGL
+export AWS_DEFAULT_REGION=us-east-1
+```
+
+- Use the control server ip for IO
+
+```
+export DATA_IP=`kubectl get svc | grep cortx-server-loadbal-svc | head -1 | awk '{print $3}'`
+```
+
+- Use the AWS CLI for IO. 
+
+```
+aws --no-verify-ssl --endpoint-url http://$DATA_IP:80 s3 ls
+aws --no-verify-ssl --endpoint-url http://$DATA_IP:80 s3 mb s3://cortx-minukube-works
+
+dd if=/dev/zero of=minikubetest bs=1M count=10
+aws --no-verify-ssl --endpoint-url http://$DATA_IP:80 s3 cp minikubetest s3://cortx-minukube-works
+aws --no-verify-ssl --endpoint-url http://$DATA_IP:80 s3 ls
+```
+
+
+### Tested by:
+
+Feb 23, 2022: Sayed Alfhad Shah(fahadshah2411@gmail.com), Rinku Kothiya(rinku.kothiya@seagate.com) and Rose Wambui(rose.wambui@seagate.com)
```

Comment:
As @osowski pointed out, it would be good to include more specifics about the version that was used. Both here and where @osowski requested it.